build/teamcity-release-upload.sh

```diff
@@ -22,6 +22,10 @@ esac
 export VERSION
 echo "Deploying $VERSION..."
 build/builder.sh build/build-static-binaries.sh static-tests.tar.gz
+for archive in cockroach-latest "cockroach-$VERSION"
```

Comment:
tiny nit: use `${var}` here and on the next line. It makes things clearer, especially when using constructions like `${archive}.src.tgz`